js/webpack.config.js

```diff
@@ -1,10 +1,18 @@
 var path = require("path");
 var version = require("./package.json").version;
 
+var mode = "production"
+var optimization = {
```

Comment:
For my own understanding is this for future use?

Reply:
yeah in order to stop webpack from complaining,  some amount of optimization will need to be specified. also setting mode to `"development"` reduces iterative build time